adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/AlignmentRecordRDDFunctions.scala

```diff
@@ -193,7 +193,7 @@ class AlignmentRecordRDDFunctions(rdd: RDD[AlignmentRecord])
       rgMetadata)
 
     // save rdd itself as parquet
-    rdd.saveAsParquet(args: ADAMSaveAnyArgs)
+    rdd.saveAsParquet(args)
```

Comment:
this can only delegate to `ADAMRDDFunctions`, like you want, since the RDD → AlignmentRecordRDDFunctions implicit is no longer being imported.